### DISCUSSION TITLE
Links In List view

### DISCUSSION THREAD

**Discussion by Shea766** [netbox-community/netbox#9008] [Q&A] [answered]
I'm having trouble getting links in a list view on my plugin. The attached picture shows the table with no links for the elements. 
I have another plugin that is working fine. I don't see a difference between the two. 

Thank you

![links](https://user-images.githubusercontent.com/71276359/161139071-5ba7fde4-22cc-4bd8-8303-1bf899b8983a.PNG)
.

**Comment by jeremystretch** [comment, marked as answer]
Can you share your column definition? It should look something like this:

```python
name = tables.Column(
    linkify=True
)
```

**Reply by Shea766** [reply]
Thanks. you pointed me right to it.
